torch/_inductor/wrapper_benchmark.py

```diff
@@ -10,6 +10,8 @@
     "pointwise",
     "reduction",
     "persistent_reduction",
+    "template",
```

Comment:
these changes are to add a bit more detail to the descriptive kernel names.

Now template and foreach kernel names will be `triton_tem_*` or `triton_for_*` instead of `triton_unk_*`